tests/test-altchains.conf

```diff
@@ -199,22 +199,6 @@
 -c ./certs/intermediate/client-chain-alt.pem
 -C
 
-# server TLSv1.2 ECDHE-ECDSA-AES128-GCM-SHA256 ECC Alt Chain
```

Comment:
Can you review why this valid test was removed?

Reply:
> Can you review why this valid test was removed?

So, when I build like this: 
`./configure --disable-rsa --disable-dh --disable-pkcs12 --enable-compkey --enable-opensslextra`

The cert chain contains RSA in higher up.  This test fails because the RSA OID is unrecognized which is because RSA is disabled.  I'm not sure how to only disable this test if RSA is disabled.

Reply:
I don't see RSA in the chain. Can you show me where it is?

Reply:
The fourth certificate in `certs/intermediate/server-chain-alt-ecc.pem` is a self signed root CA with an RSA key and signature.  I suppose with altcertchain enabled, its reading this certificate, encountering an OID it does not understand since RSA is disabled and then returning unknown OID error.  

What confuses me is why this wasn't happening before my change.